tests/driver/api.test.ts

```diff
@@ -470,8 +470,8 @@ describe('Mongoose Model API level tests', async () => {
             ).then(() => null, err => err);
             assert.deepStrictEqual(err.status.documentResponses, [
                 { _id: '0'.repeat(24), status: 'ERROR', errorsIdx: 0 },
-                { _id: '1'.repeat(24), status: 'ERROR', errorsIdx: 0 },
-                { _id: '2'.repeat(24), status: 'ERROR', errorsIdx: 0 }
+                { _id: '1'.repeat(24), status: 'ERROR', errorsIdx: 1 },
+                { _id: '2'.repeat(24), status: 'ERROR', errorsIdx: 2 }
```

Comment:
Is this because of any change on the Data API server response? (i.e. the errorsIdx value changed now?)

Reply:
Correct, this is a change on the Data API side. From my understanding, the reason is that unique key errors are now different because they contain the duplicate value, so now there are 3 distinct errors rather than 1.